examples/simple-game-server/Makefile

```diff
@@ -87,13 +87,9 @@ endif
 ifeq ($(WITH_ARM64), 1)
 push: push-linux-image-arm64
 endif
-	-docker manifest rm $(server_tag)
```

Comment:
Just noting we'll want to increment the version number as well in here, since this will be a new build (then we will want to update the simple-game-server version everywhere as well - likely in a separate PR).